packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -37,8 +38,12 @@ const renderGraph = (props: {
     }
 };
 
+export interface HasUserInput {
+    getUserInput(): PerseusGraphType
+}
+
 export const MafsGraph = React.forwardRef<
-    Partial<Widget>,
```

Comment:
The intent of using the Widget type, which was @jeremywiebe's request, was that we would eventually implement the other methods (focus, blur, etc) on this component. This change makes that more difficult in the future.

Reply:
I've changed this back to use `Partial<Widget>` I think we lose some type safety by doing that (the Widget type uses `any` for some fields) but it's not a big deal.